docs/reference/cluster/tasks.asciidoc

```diff
@@ -79,8 +79,58 @@ GET _tasks?parent_task_id=parentTaskId:1 <1>
 
 <1> This won't return a 404 if the parent isn't found.
 
-The task API can be also used to wait for completion of a particular task. The following call will
-block for 10 seconds or until the task with id `oTUltX4IQMOUUVeiohTt8A:12345` is completed.
+You can also use the `detailed` request parameter to get more information about
+the running tasks. This is useful for telling one task from another but is more
+costly to execute. For example, fetching all searches using the `detailed`
+request parameter:
+
+[source,js]
+--------------------------------------------------
+GET _tasks?actions=*search&detailed
```

Comment:
Do you want to make this explicit with `detailed=true` rather than the implicit `detailed`?  Up to you, but I know when I first got started using ES the implicit true seemed like voodoo to me